DolbyIO/Source/Public/DolbyIOObserver.h

```diff
@@ -62,7 +62,7 @@ class DOLBYIO_API UDolbyIOObserver : public UActorComponent
 	UPROPERTY(BlueprintAssignable, Category = "Dolby.io Comms")
 	FObserverOnParticipantAddedDelegate OnParticipantAdded;
 
-	/** Triggered when a remote participant leaves the conference. */
+	/** Triggered when a remote participant's status in the conference is changed. */
```

Comment:
"Triggered when a remote participant's status is updated." should be enough (and more technically accurate since on_air -> on_air is not a change :P).